enclave/core/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-if (UNIX)
+if (UNIX OR USE_CLANGW)
```

Comment:
This is where I got confused trying to reason out our option logic. I think you're right, it makes sense to leave this as `UNIX OR CLANG_ON_WINDOWS`. When I replaced it with `BUILD_ENCLAVES` it just didn't make sense in the context.

Reply:
Can you look at the updated PR and see if it resonates?